workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -118,36 +121,41 @@ describe('CostCenterService', () => {
 
   describe('create', () => {
     describe('with a valid CreateCostCenter object', () => {
-      const accountId = `${resourceTypeToKey.account.toLowerCase()}-sampleAccId`;
-      const createCostCenter: CreateCostCenter = {
-        name: 'the name',
-        description: 'the description',
-        accountId: accountId
-      };
+      let accountId: string;
+      let createCostCenter: CreateCostCenter;
+
+      beforeEach(() => {
+        accountId = `${resourceTypeToKey.account.toLowerCase()}-sampleAccId`;
+        createCostCenter = {
+          name: 'the name',
+          description: 'the description',
+          accountId: accountId
+        };
+      });
 
       describe('`dependency` is the id of a saved Account', () => {
         beforeEach(() => {
           ddbMock.on(GetItemCommand).resolves({
-            Item: marshall(accountMetadata, {
+            Item: marshall(account, {
               removeUndefinedValues: true
             })
           });
-
+          accountId = 'whatever';
```

Comment:
lol, interesting choice. Why don't we just keep what it originally was?
`${resourceTypeToKey.account.toLowerCase()}-sampleAccId`;`